### DISCUSSION TITLE
How to install the better-xcloud.user.js on Edge

### DISCUSSION THREAD

**Discussion by NMeckbach** [redphx/better-xcloud#258] [General]
Hello guys,
I want to use better xcloud on my Steam Deck using Edge.
I have installed the tampermonkey extension and downloaded the JavaScript file.
What do I have to do with this JavaScript file?
I really don't understand.
Could someone help me pls?
Thanks in advance.

**Comment by redphx** [comment]
Open this link https://github.com/redphx/better-xcloud/releases/latest/download/better-xcloud.user.js and you'll have an option to install the script inside the Tampermonkey extension
